Apps/Common/src/Auditing/AuditLogger.cs

```diff
@@ -35,13 +35,14 @@ namespace HealthGateway.Common.Auditing
     /// </summary>
     public class AuditLogger : IAuditLogger
     {
+        private const string TESTHOST_NAME = "testhost";
         private readonly ILogger<IAuditLogger> logger;
 
         private readonly IConfiguration configuration;
 
-        private readonly AuditDbContext dbContext;
+        private readonly IAuditDbContext dbContext;
 
-        public AuditLogger(ILogger<IAuditLogger> logger, AuditDbContext dbContext, IConfiguration config)
+        public AuditLogger(ILogger<IAuditLogger> logger, IAuditDbContext dbContext, IConfiguration config)
```

Comment:
instead of passing an interface for the context, lets use a delegate to do work on the DB. That way we can set the result to whatever we need